app/services/record_declarations/base.rb

```diff
@@ -64,10 +83,11 @@ def create_declaration_attempt!
     end
 
     def find_or_create_record!
-      self.class.declaration_model.find_or_create_by!(declaration_parameters).tap do |participant_declaration|
-        DeclarationState.submitted!(participant_declaration)
-        participant_declaration.make_eligible! if fundable?
-      end
+      self.class.declaration_model.find_or_create_by!(declaration_parameters)
```

Comment:
This should still `DeclarationState.submitted!(participant_declaration)`

Reply:
this has been moved further up into `#call` 
https://github.com/DFE-Digital/early-careers-framework/pull/1605/files#diff-ec96a6b08aefdf2afd683dd457777d9d18ca2029614ad570f21bb4b25f0df667R44